### ISSUE TITLE
update pdfbox

### ISSUE THREAD

**Issue by h908714124** [GlenKPeterson/PdfLayoutManager#17]
it's currently using `1.8.10`, already 2 years behind

**Comment by GlenKPeterson** [comment]
Thank you for encouraging this update.  PdfLayoutManager 0.5.0 is just released with PDFBox 2.0.6.  
I think that outside of character encoding things look like they are working correctly.  But the character encoding is a bit of an unknown.  I have not updated documentation for 0.5.0 yet.

0.5.0 handles characters differently.  I think the character encoding issues in an old (even older than 1.8.10) version of PDFBox have been fixed.  So I removed my Transliteration code for Russian which also converted any "High ANSI characters" to lower ANSI equivalents, or to bullets if no equivalent is available.  If you get exceptions about character support, you now need to load a font that includes those characters, or trap them yourself.  Here's how to load a font: https://pdfbox.apache.org/1.8/cookbook/workingwithfonts.html

Here's how I used to trap them (incomplete with the new version of PDFBox):
https://github.com/GlenKPeterson/PdfLayoutManager/blob/master/src/main/java/com/planbase/pdf/layoutmanager/PdfLayoutMgr.java#L892
